pkgs/servers/search/manticoresearch/default.nix

```diff
@@ -0,0 +1,90 @@
+{ lib, stdenv, fetchFromGitHub, fetchurl
+, bison, cmake, flex
+, boost, icu, mariadb-connector-c, re2
+}:
+let
+  boost_static = boost.override { enableStatic = true; };
```

Comment:
can we use `boost = boost.override { enableStatic = true; };` on all-packages.nix?

Reply:
OK. (I also want to re-check if this is still even necessary with all the changes that have happened)